### DISCUSSION TITLE
zk logo ideas?

### DISCUSSION THREAD

**Discussion by tjex** [zk-org/zk#415] [General]
`zk` deserves a logo. The auto-generated pixel art thing has had its run, I reckon.

It would be used anywhere and everywhere (atm just GitHub, website (https://github.com/zk-org/zk/discussions/388)). 

If anyone is interested to contribute some designs, or just float some ideas, comment away! @zk-org/maintainers ?
In keeping with our open source nature, any logo designs would need to be supplied under the [creative commons 4.0](https://creativecommons.org/licenses/by/4.0/).

Size should be min 300x300 and `png`.

Otherwise, I had a quick go and if people are happy, we could use any of these, or variations thereof. Logos with borders would be more for websites or places where it needs to float by itself, in which case the border could help it stand out:

<img width="300" alt="zk-modern" src="https://github.com/zk-org/zk/assets/83029642/e45c901a-70dc-4aff-b9e9-163dfb507a0e">

---

<img width="300" alt="zk-white-modern" src="https://github.com/zk-org/zk/assets/83029642/18a3bf6c-f37a-4e4a-b63a-1e9acab72638">

---

<img width="300" alt="zk-black" src="https://github.com/zk-org/zk/assets/83029642/01e610aa-c6a0-49a3-a857-621cc8ec7a8c">

---

<img width="300" alt="zk-white" src="https://github.com/zk-org/zk/assets/83029642/11bca5e4-bacb-4988-81a3-4959e5a3be19">

---

<img width="300" alt="zk-black-solid-border" src="https://github.com/zk-org/zk/assets/83029642/b5f050d1-140d-4e57-a499-19c7b9ae660c">

---

<img width="300" alt="zk-white-dashed-border" src="https://github.com/zk-org/zk/assets/83029642/bb1e267d-185f-4004-bd13-35409ffb869e">

---

**Comment by mickael-menu** [comment]
In favor of the "modern" variant (first ones) 👍

**Comment by mcDevnagh** [comment]
I'm the worst at stuff like this. They all look good to me

**Comment by jurica** [comment]
I like the first two ones the most as well.
